compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -4973,8 +4973,8 @@
                     "type": "Token"
                 },
                 {
-                    "name": "errorConstructor",
-                    "type": "ErrorConstructorExpressionNode"
+                    "name": "expression",
+                    "type": "ExpressionNode"
```

Comment:
We need to do the same for `api_gen_syntax_tree_descriptor.json` which is used by the shell.